.eslintrc.cjs

```diff
@@ -8,5 +8,6 @@ module.exports = {
   rules: {
     'no-console': 0,
     '@typescript-eslint/naming-convention': 0,
+    'import/no-extraneous-dependencies': 0
```

Comment:
Is it safe to disable this check? I wonder if it introduces any risks due to us importing implicit dependencies.

Reply:
You're right, it's better to be safe here. I've changed this to keep it enabled.